src/senaite/sync/browser/views.py

```diff
@@ -499,6 +524,26 @@ def get_storage(self, domain=None):
             self.storage[domain]["credentials"] = OOBTree()
         return self.storage[domain]
 
+    def reindex_updated_objects(self):
```

Comment:
Please change the signature to `reindex_uids(self, uids)` and change the method call accordingly

Reply:
So this method will take 2 arguments where one (__uids__) is already saved in `self`?

Reply:
I always prefer explicit over implicit coding. Although it works perfectly, it is not obvious at a first glance which steps need to run first to fill that `self.uids` list